search/filters/SearchFilter.php

```diff
@@ -178,7 +183,7 @@ public function getDbName() {
 			return '"' . implode('"."', $parts) . '"';
 		}
 		
-		return "\"$candidateClass\".\"$this->name\"";
+		return "\"{$candidateClass}\".\"{$this->name}\"";
```

Comment:
Well, $var replacement doesn't work with single quotes, so I've replaced this line and used a sprintf instead. Sound better?